src/signals/incident-management/containers/IncidentDetail/components/DetailHeader/index.js

```diff
@@ -6,38 +6,56 @@ import DownloadButton from './components/DownloadButton';
 
 import './style.scss';
 
-const DetailHeader = ({ incident, baseUrl, onThor, accessToken }) => {
+const DetailHeader = ({ incident, baseUrl, accessToken, onPatchIncident }) => {
   const status = incident && incident.status && incident.status.state;
-  const canSplit = (status === 'm') && !(incident && incident.parent_id);
+  const canSplit = (status === 'm') && !(incident && (incident._links['sia:children'] || incident._links['sia:parent']));
```

Comment:
Why was this change necessary?

Reply:
er werd gecheckt op verkeerde voorwaarden: de parent en children zitten niet meer in incident maar in de _links['sia:children'] en _links['sia:parent']

je mag niet kunnen splitsen als de incident niet al eerder gesplitst is.